pages.es/linux/turbostat.md

```diff
@@ -0,0 +1,24 @@
+# turbostat
+
+> Informa de la topología del procesador, frecuencia, temperatura, potencia y estadísticas de inactividad.
+> Más información: <https://manned.org/turbostat.8>.
+
+- Muestra las estadísticas cada 5 segundos:
+
+`sudo turbostat`
+
+- Muestra las estadísticas cada determinado número de segundos:
```

Comment:
```suggestion
- Muestra las estadísticas cada número de segundos:
```

Reply:
- Muestra las estadística cada un determinado número de segundos:

Reply:
The word `estadística` is not in plural, how about `cierto` instead of `determinado` @kant?

Reply:
Sorry, there is a typo: `estadísticas` (my bad!) So, imho, it should be:
* Muestra las estadísticas cada un cierto número de segundos:

Reply:
Without the article `un` sounds good to me, does it to you?

Reply:
Perfect!

Reply:
I'll update the suggestion to match our agreed solution and apply it

Reply:
updated!